overlord/snapstate/autorefresh.go

```diff
@@ -1025,11 +1025,18 @@ func computeSnapRefreshFailureSeverity(chg *state.Change, unlinkTask *state.Task
 		if snapsup.InstanceName() != snapName {
 			continue
 		}
-		// Refresh failure happened after a reboot
-		return snap.RefreshFailureSeverityAfterReboot
+
+		bootBase, err := deviceModelBootBase(chg.State(), nil)
```

Comment:
Nitpick: In practical terms this will never happen currently, but determing the bootBase could be before the loop to avoid potentionally avoid looking it more than once.

Reply:
updated, thank you!